src/main/java/pl/bugdemons/ui/selenide/CustomSelenideCommands.java

```diff
@@ -0,0 +1,27 @@
+package pl.bugdemons.ui.selenide;
+
+import com.codeborne.selenide.commands.Commands;
+import com.codeborne.selenide.impl.ElementFinder;
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.By;
+
+import pl.bugdemons.ui.selenide.customcommand.SetValueWithEnterCommand;
+import pl.bugdemons.ui.selenide.customcommand.SetValueWithTabCommand;
+
+import static com.codeborne.selenide.WebDriverRunner.driver;
+
+@Slf4j
+public class CustomSelenideCommands {
+
+    private CustomSelenideCommands() {
+    }
+
+    public static void setUpSelenideCustomCommands() {
```

Comment:
jak już to wszystko ma być przykładem jak to zaimplementować to warto byłoby dodać doc stringi z wyjaśnieniem co do czego służy. Co które parametry robią i po co są elementy. Takie troche wiki by powstało